src/main/java/edu/umich/its/cpm/EmailFormatter.java

```diff
@@ -157,5 +270,11 @@ public String subStringFrom(String emailText, String contentToChopFrom) {
         return emailText.substring(posA);
     }
 
+    public String subStringInBetween(String emailText, String chopper) {
+        String choppingFirstBoundary = emailText.substring(chopper.length());
```

Comment:
Yes, and yes the name don't make sense. I have renamed it. I have a unit test written to just to test this part of the code